tools/extras/install_irstlm.sh

```diff
@@ -55,6 +55,7 @@ fi
   [ ! -z ${IRSTLM} ] && \
     echo >&2 "IRSTLM config is already in env.sh" && exit
 
+  wd=`pwd`
   wd=`readlink -f $wd 2>/dev/null || pwd`
```

Comment:
oh I see.  you could probably replace $wd in this expression with $PWD, and remove the extra line.